src/openssl/cipher.cr

```diff
@@ -1,6 +1,41 @@
 require "random/secure"
 require "openssl"
 
+# A class which can be used to encrypt and decrypt string data using a specified cipher.
```

Comment:
It's not only for string, is it?

Reply:
Sorry I wasn't sure what is allowed for input into `OpenSSL::Cipher#update`. I'm guessing it's anything that can be turned into `Bytes`, right?

Reply:
Yeah the only technical requirement seems to be having a `#to_slice` method. The practical idea is obviously to allow `String | Bytes`. Maybe that should be added as a type restriction to communicate this clearly. This would prevent you from passing any other type that implements `#to_slice` but that's not common and you can just convert it to a slice at the callsite.